sources/packages/backend/workflow/BPMN/camunda-8/fulltime-configure-disbursement.bpmn

```diff
@@ -61,14 +61,14 @@
       <bpmn:incoming>Flow_1bx16ip</bpmn:incoming>
       <bpmn:outgoing>Flow_1kesibu</bpmn:outgoing>
     </bpmn:intermediateThrowEvent>
-    <bpmn:intermediateThrowEvent id="Event_0aunmqb" name="Configure Payload">
+    <bpmn:intermediateThrowEvent id="Event_0aunmqb" name="Configure Payload for 18 weeks">
```

Comment:
For the 18 weeks, the second disbursement will contain only `Canada Grants`. Is it possible that none of those grants would be eligible making the second disbursement in the payload as an empty array?
Does the worker's code need any adjustment to deal with it?

Reply:
@andrewsignori-aot I tested the above, I believe we are already taking care of it, I created and application of 18 weeks which is not eligible for any grants, below are my findings,
assessment flow,
![image](https://github.com/bcgov/SIMS/assets/77353155/b4c9ea9c-da7c-4397-8dff-3e49e8aca1f8)
`disbursement_schedules` will have 2 records, 
![image](https://github.com/bcgov/SIMS/assets/77353155/fea3b006-70f5-4055-a27d-9865132284a9)
but `disbursement_values` will have only eligible disbursement award i.e. disbursement-1 award,
![image](https://github.com/bcgov/SIMS/assets/77353155/4014e059-4d30-4b66-b411-1a88f35e0f94)
And NOA will have only 1 disbursement ,
![image](https://github.com/bcgov/SIMS/assets/77353155/e8eacfb1-596c-4a48-990a-f688739f6b72)

so should we still need to restrict it in camunda level?

Reply:
updated as suggested